doc/man3/SSL_CTX_new.pod

```diff
@@ -73,13 +73,13 @@ functions
 
 =head1 DESCRIPTION
 
-SSL_CTX_new_ex() creates a new B<SSL_CTX> object as a framework to
-establish TLS/SSL or DTLS enabled connections using the library context
-I<libctx> (see L<OSSL_LIB_CTX(3)>). Any cryptographic algorithms that are used
-by any B<SSL> objects created from this B<SSL_CTX> will be fetched from the
-I<libctx> using the property query string I<propq> (see
-L<provider(7)/Fetching algorithms>. Either or both the I<libctx> or I<propq>
-parameters may be NULL.
+SSL_CTX_new_ex() creates a new B<SSL_CTX> object, which stores various options
```

Comment:
```suggestion
SSL_CTX_new_ex() creates a new B<SSL_CTX> object, which holds various configuration and data
```

The word "option" might imply the literal configuration options like `SSL_OP_NO_RENEGOTIATION` but the `SSL_CTX` holds function pointers for callback functions, as well as statistics about incoming connections.